keras/callbacks.py

```diff
@@ -1971,11 +1971,11 @@ class EarlyStopping(Callback):
           displays messages when the callback takes an action.
       mode: One of `{"auto", "min", "max"}`. In `min` mode,
           training will stop when the quantity
-          monitored has stopped decreasing; in `"max"`
-          mode it will stop when the quantity
-          monitored has stopped increasing; in `"auto"`
-          mode, the direction is automatically inferred
-          from the name of the monitored quantity.
```

Comment:
Keep the sentence "the direction is automatically inferred from the name of the monitored quantity." The content that follows provides added information but the sentence is still useful.

Reply:
Do I have to click "Resolve conversation" or does someone else do this?